sdk/purview/purview-administration-rest/samples-dev/accountCollections.ts

```diff
@@ -18,15 +18,15 @@ const endpoint = process.env["ENDPOINT"] || "";
 
 async function main(): Promise<void> {
   console.log("== List collections sample ==");
-  const client = PurviewAccountClient(endpoint, new DefaultAzureCredential());
+  const client = PurviewAccount.createClient(endpoint, new DefaultAzureCredential());
```

Comment:
this is a multi client case, the previous code is manually written before codegen implemented it and when we implement multi-client, we have discussed the user experience and we want something like `PurviewAccount.createClient`.